README.md

```diff
@@ -56,6 +56,7 @@ some text
     - [Nginx](#nginx)
   - [Contributing](#contributing)
     - [License](#license)
+- [3rd party clients](#3rd-party-clients)
```

Comment:
I'd prefer to have this before the Contributing section.

Reply:
Hmm, ok. I didn't think that the 3rd party clients section would fit under `Usage`.